static/js/pages/instances/InstanceList.tsx

```diff
@@ -42,6 +73,34 @@ const InstanceList: FC = () => {
   const [query, setQuery] = useState<string>("");
   const [status, setStatus] = useState<string>("any");
   const [type, setType] = useState<string>("any");
+  const [userHidden, setUserHidden] = useState<string[]>(loadHidden());
+  const [sideHidden, setSideHidden] = useState<string[]>([]);
+
+  const hidden = userHidden.concat(sideHidden);
```

Comment:
I would expect this to go after L98 since that call to `resizeColsForSidepanel` might update the `sideHidden` state.

Reply:
It shouldn't matter, because there will be another render cycle after the resize function updates the state. so even if it is listed after the function, it will not receive the updated values on the 1st render.

I put it there, so it is clear for the code reader how the two states are just combined.